pkg/operatorsource/configuring_test.go

```diff
@@ -225,13 +228,6 @@ func TestReconcile_CatalogSourceConfigAlreadyExists_Updated(t *testing.T) {
 	ctx := context.TODO()
 	opsrcIn := helperNewOperatorSourceWithPhase(namespace, name, phase.Configuring)
 
-	labelsWant := map[string]string{
-		"opsrc-group":                     "Community",
-		builders.OpsrcOwnerNameLabel:      "foo",
-		builders.OpsrcOwnerNamespaceLabel: "marketplace",
-	}
-	opsrcIn.SetLabels(labelsWant)
```

Comment:
I see why you shouldn't be setting the object with the expected labels but I don't see where you are checking if the expected labels are present.

Reply:
Yeah it doesn't seem like I am checking the labels anywhere - good catch!

Reply:
After looking at this in more detail, I don't think this is possible for unit tests. Since these are just mock calls, child resources aren't actually getting created so how can we check that labels are correctly set?

Reply:
This is not for the child resources but the object itself. I think you check the labels between `opsrcIn` and `opsrcGot`. Maybe that is why the labels were being set where you are removing it. So maybe you should not get rid of this code. The `assert.Equal()` might already be taking care of this check but please confirm.

Reply:
the `assert.Equal` here does check that the labels set before calling the reconciler are still there after the reconciler is called. This is orthogonal to the owner labels however. I will add the first label back but it seems misleading to add the second and third labels (lines 230 and 231).

In order to have unit tests for the owner labels I would need to add unit tests in the grpccatalog pkg and the registry pkg where the child resources are created. I would argue that this is redundant because the e2e tests now check for this label propagation. But I am happy to do this if the team thinks this is crucial.